src/api/local/deployment.ts

```diff
@@ -479,24 +479,28 @@ export namespace Deployment {
   }
 
   async function setDeployLocation(node: any, workspaceFolder?: WorkspaceFolder) {
-    const path = node?.path || await vscode.window.showInputBox({
+    const connection = instance.getConnection();
+    const possibleDeployDir = path.posix.join(`/`, `home`, connection!.currentUser, `builds`, workspaceFolder ? workspaceFolder.name : `codebase`);
```

Comment:
How about using the actual and/or configured home directory?
```suggestion
    const possibleDeployDir = path.posix.join(connection?.config?.homeDirectory || `/home`, connection!.currentUser, `builds`, workspaceFolder ? workspaceFolder.name : `codebase`);
```

Reply:
@sebjulliand Yes. The reason we build the path is because `config.homeDirectory` is not actually the home directory all the time. It acts more like the 'working directory', and we don't want to build path to be based on the working directory.

Reply:
Sounds about right.
Now, call me paranoid, but I'd run `pwd` or `echo ~` and get the output here to get the actual home directory. I know most of the time `/home/<username>` is the actual home dir but...better safe than sorry? 😁